### DISCUSSION TITLE
Scope of Editorial Workflow configuration

### DISCUSSION THREAD

**Discussion by privatemaker** [decaporg/decap-cms#7050] [General]
Upon using Decap CMS with a site whereby there is no `publish_mode` setting present in my `config.yml` I am still getting the [Editorial Workflow](https://decapcms.org/docs/add-to-your-site/#editorial-workflow) experience of creating a branch and then merging.

Is it not possible to be disable Editorial Workflow if the backend supports it?
Or is there a default value required which is missing from the documentation?

**Comment by martinjagodic** [comment]
@privatemaker can you share your entire config? It will be easier to debug. However, if I don't have the `publish_mode`, there is no signs of Editorial Workflow.
